app/packages/components/src/components/TabOption/TabOption.tsx

```diff
@@ -77,9 +78,9 @@ const TabOption = ({
       onMouseEnter={() => set({ background: theme.background.body })}
       onMouseLeave={() => set({ background: theme.background.level1 })}
     >
-      {options.map(({ text, title, onClick }, i) => (
+      {options.map(({ text, title, onClick, dataCy }, i) => (
         <Tab
-          data-cy={`tab-option-${title}`}
+          data-cy={dataCy}
```

Comment:
I think this change is breaking existing tests. Easy fix is make `tab-option-${title}` the fallback if `dataCy` is not provided
```suggestion
          data-cy={dataCy || `tab-option-${title}`}
```